src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
When Im reading your comments, I feel like you are taking good care about the *implementation* side of the interface, not the *user* side. 

I do agree with your points and your suggestion would be a good solution you consider existing PSR-7 implementations and what the "best solution" would be. But as stated, let's not overlook the developer experience. 

I do not have any more arguments that is not already said. Now, after this long (and interesting) discussion, could you write a (non-inline) comment what your current suggestion for stream factories are?